src/data_aggregator.ts

```diff
@@ -112,6 +112,8 @@ export interface DataAggregatorConfig {
    * The minimum aggregate volume across all exchanges to report
    */
   minAggregatedVolume: BigNumber
+
+  devMode: boolean
```

Comment:
Please consider adding a comment, for consistency with the other config parameters.